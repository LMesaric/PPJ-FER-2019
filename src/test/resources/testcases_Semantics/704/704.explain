semantic: y already declared
note: 'y' on line 3
