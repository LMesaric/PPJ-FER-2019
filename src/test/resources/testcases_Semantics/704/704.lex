KR_INT 1 int
IDN 1 main2
L_ZAGRADA 1 (
KR_VOID 1 void
D_ZAGRADA 1 )
L_VIT_ZAGRADA 1 {
KR_INT 2 int
IDN 2 x
OP_PRIDRUZI 2 =
BROJ 2 5
ZAREZ 2 ,
IDN 2 y
TOCKAZAREZ 2 ;
KR_INT 3 int
IDN 3 y
TOCKAZAREZ 3 ;
KR_RETURN 4 return
BROJ 4 0
TOCKAZAREZ 4 ;
D_VIT_ZAGRADA 5 }
