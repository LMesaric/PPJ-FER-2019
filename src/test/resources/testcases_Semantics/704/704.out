<izravni_deklarator> ::= IDN(3,y)
